### PR TITLE
Bug 1698835 - Integrate PollyJs into Perfherder

### PR DESCRIPTION
**Pull request by esanuandra**

### CODE REVIEW

File: tests/ui/integration/graphs-view/graphs_view_integration_test.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+describe('GraphsViewRecord Test Pupeteer', () => {
+  const context = setupPolly({
+    adapters: ['puppeteer'],
+    adapterOptions: {
+      puppeteer: { page },
+    },
+    persister: 'fs',
+    persisterOptions: {
+      fs: {
+        recordingsDir: path.resolve(__dirname, 'recordings'),
```
Comment (airimovici) [inline tests/ui/integration/graphs-view/graphs_view_integration_test.jsx:20]: recordings folder should be directly in /integration

**Comment by airimovici** [comment]
We need to also run "npm install" besides "yarn install" for puppeteer to have it's dependencies. This would be a problem when we want to run the tests in circleCI

**Review by beatrice-acasandrei — APPROVED** [review]
Looks good now!

**Comment by alexandru-io** [comment]
Tests on linux look good.

File: docs/testing.md [hunk 2]
```diff
@@ -55,6 +55,15 @@ yarn test:watch
...
 The tests will perform an initial run and then re-execute each time a project file is changed.
 
+### Running the Jest front-end integration tests
+
+The integration tests for the UI are run with [Jest] and [Puppeteer].
+
+To run the tests:
+
+- If you haven't already done so, install local dependencies by running `yarn install` from the project root.
+- Then run `yarn test:integration` to execute the tests.
```
Comment (sarah-clements) [inline docs/testing.md:65]: This reads like a duplicate of the above section. Can you combine these two sections and mention that tests are written with react testing library and that Polly can be used to mock APIs (maybe some info about the best time to use it over using fetch mock?).

Comment (sarah-clements) [reply docs/testing.md]: Thank you, that's much clearer.

**Review by sarah-clements — APPROVED** [review]
Looks good - thank you. @airimovici you can merge this pr once all the tests pass.

**Review by airimovici — APPROVED** [review]

**Pull request merged**